src/query/src/parser.rs

```diff
@@ -168,7 +168,6 @@ impl QueryLanguageParser {
     }
 
     fn parse_promql_timestamp(timestamp: &str) -> Result<SystemTime> {
-        // FIXME(dennis): aware of timezone
         // try rfc3339 format
```

Comment:
Is this todo intentionally removed so that promql timestamp literal parsing does not consider time zone?

Reply:
The `rfc3339` format contains the timezone info, so we don't need to this fix.